### PR TITLE
Css task 5

### PR DESCRIPTION
**Pull request by dudzon**
https://dudzon.github.io/ui-internship-lwo2/css_tasks/task5/

### CODE REVIEW

File: css_tasks/task5/index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    />
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <!-- INPUTS TO CHANGE DEFAULT STYLES -->
+
+    <div class="container">
+    <label for="theme-toggler" class="container__toggler toggler--theme">Change Theme</label>
+    <input type="checkbox" id="theme-toggler" class="theme-toggler">
+    <label for="font-toggler" class="container__toggler toggler--font">Change Font</label>
```
Comment (tkachukivan) [inline css_tasks/task5/index.html:20]: by BEM you can't have modifier class if you don't have base class, so you need to write is just as class with one `-` or add base class which is toggler

File: css_tasks/task5/main.css [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+  -webkit-appearance: none;
+}
+/**********************
+*                     *
+*      HEADER         *
+*                     *
+***********************/
+.header {
+  width: 100%;
+  height: 7rem;
```
Comment (tkachukivan) [inline css_tasks/task5/main.css]: generally it is better not to specify height of elements, let your content to do this(or sometimes do it by paddings), you can remove it with margin-top on content__main-section and everything will looks fine

File: css_tasks/task5/main.css [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+  margin-left: 20px;
+  left: 0;
+  cursor: pointer;
+  padding: 5px;
+  border: 1px solid var(--fontColor);
+  border-radius: 5px;
+  width:100px;
+  font-family: var(--mainThemeFont);
+  font-size:.77rem;
+  text-align:center
```
Comment (tkachukivan) [inline css_tasks/task5/main.css]: you still write css properties in completely chaotic order(you have a lot of them in chaotic order in file), try to have some and follow it for consistency reasons and always!!! put `;` after your property

```css
.container__toggler {
  position: fixed;
  left: 0;
  width:100px;
  margin-left: 20px;
  padding: 5px;
  border: 1px solid var(--fontColor);
  border-radius: 5px;
  font-family: var(--mainThemeFont);
  font-size:.77rem;
  text-align:center;
  cursor: pointer;
}
```

File: css_tasks/task5/main.css [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+  width: 100%;
+  height: 100vh;
+}
+/* MAIN SECTION */
+.content__main-section {
+  font-family: var(--mainThemeFont);
+  margin-top: 4rem;
+}
+.content__article {
+  width: 100%;
```
Comment (tkachukivan) [inline css_tasks/task5/main.css:105]: block elements by default have 100% width

File: css_tasks/task5/main.css [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+*                     *
+***********************/
+
+/* Firefox fix for main article header */
+@supports not (column-span:all){
+  .content__article{
+    margin-top:10rem;
+  }
+  .article__header{
+    position:absolute;
```
Comment (tkachukivan) [inline css_tasks/task5/main.css]: I understand that it is fix, but I think that there is other way to do it without absolute positioning, as it looks very strange

**Review by tkachukivan — APPROVED** [review]
Read comments and try to follow some order when write css properties, it is a bit hard to read code when you can find display property in beginning, in the middle, in the end, you all the time have to read code attentively and if it can be fine with small code base, but with big one it is become really hard

File: css_tasks/task5/index.html (hunk 1 above)
Comment (dudzon) [reply css_tasks/task5/index.html:20]: ok

File: css_tasks/task5/main.css (hunk 5 above)
Comment (dudzon) [reply css_tasks/task5/main.css]: probably yes, but I think I would have to change HTML structure.

**Review by tsvetkovv — CHANGES_REQUESTED** [review]
- fix image width
- fix header line height
image


- fix section margin (it should not overlap on other elements)
image


- fix header height
image

**Review by tsvetkovv — APPROVED** [review]

**Pull request merged**